devtools/composer.json

```diff
@@ -4,7 +4,7 @@
         "ext-json": "*",
         "ext-simplexml": "*",
         "apigen/apigen": "^2.8",
-        "fabpot/php-cs-fixer": "^1.11",
+        "friendsofphp/php-cs-fixer": "^1.11,<1.11.5",
```

Comment:
Thanks, 

This PR goal is "Add PHP7.1 for TravisCI" .
So I made a little correction.
Actually, preparation of migration is done little by little.

branch:
> https://github.com/KazuakiM/php-coveralls/tree/upgradePhpCs

TravisCI:
> https://travis-ci.org/KazuakiM/php-coveralls/builds/200877621